docs/source/installation/self_host.rst

```diff
@@ -67,7 +67,39 @@ Now to get some data into the application, you should follow the
 Customizing your deployment (optional)
 --------------------------------------
 
-By default, the docker-compose.yml file that gets run comes with a series of default config values that should work for
-most deployments. If you need to customize anything, you can take a look at the full list of
+The configuration file exists under the name `oss-conf.env` and comes with a series of default config values that should
+work for most deployments. If you need to customize anything, you can take a look at the full list of
 :doc:`environment variables </installation/configuration>`.
 After making any changes, simply restart the stack with docker-compose.
+
+
+Define SMTP for email integration
+---------------------------------
+
+Deepchecks can send emails to users when they are invited to the platform and when there is a new alert. To enable
```

Comment:
Change to: "Deepchecks sends emails to users in multiple scenarios. Among these, it will send emails to users that are invited to the platform and can be configured to send emails when there is a new alert."